example/example.js

```diff
@@ -1,23 +1,41 @@
 const path = require("path");
 const Potrace = require("..");
 const fs = require("fs-extra");
-const Svg2 = require("oslllo-svg2");
+const { svg2png, initialize } = require('svg2png-wasm-node-10');
+const wasm = require.resolve('svg2png-wasm-node-10').replace(/(svg2png-wasm-node-10).*/, '$1/svg2png_wasm_bg.wasm')
+
+const init = async () => {
+    const buffer = await fs.readFile(wasm)
+
+    await initialize(buffer)
+}
+
+let initPromise;
+
+const draw = async (svg) => {
+    initPromise = initPromise || init()
+    await initPromise
+
+    return svg2png(svg)
```

Comment:
Add the background option to make the example output visible for dark mode users.

```js
return svg2png(svg, { backgroundColor: 'white', });
```
Current output

![image](https://user-images.githubusercontent.com/20132952/147812287-0a583735-ae1b-48ef-b94b-b6806acc31b1.png)